src/diagonal.works/b6/world.go

```diff
@@ -15,17 +15,38 @@ import (
 	"github.com/golang/geo/s2"
 )
 
-// TODO: harmonise Value and Literal in expression.go
+// TODO(mari): harmonise Value and Literal in expression.go
 type ValueType int
 
 const (
 	ValueTypeString ValueType = iota
 	ValueTypeLatLng
+	ValueTypeValues
```

Comment:
In the spirit of harmonising Literal and Value, I feel this should be ValueTypePath, rather than adding a new array type (since we don't actually have an array Literal)

Reply:
no i refuse this is nicer
(for now, i ll do harmonize w literal in a second so let s see what comes out then)